easyfsl/methods/abstract_meta_learner.py

```diff
@@ -13,10 +15,12 @@ class AbstractMetaLearner(nn.Module):
     """
 
     def __init__(self, backbone: nn.Module):
-        super(AbstractMetaLearner, self).__init__()
+        super().__init__()
         self.backbone = backbone
         self.criterion = nn.CrossEntropyLoss()
 
+    # pylint: disable=all
+    @abstractmethod
     def forward(
```

Comment:
Disable pylint only for the `forward` function so that you don't need to enable it again line 39:
```
def forward (   # pylint: disable=...
```

See https://stackoverflow.com/questions/28829236/is-it-possible-to-ignore-one-single-specific-line-with-pylint

Reply:
It doesn't seem like it does the trick: I put a useless string further down in the file and it doesn't detect if without an enable